python/pyspark/mllib/tests.py

```diff
@@ -75,22 +75,23 @@
     # No SciPy, but that's okay, we'll skip those tests
     pass
 
-ser = PickleSerializer()
```

Comment:
What do you mean by concurrency errors? AFAIK we run tests serially in these files.